trin-core/src/portalnet/storage.rs

```diff
@@ -131,7 +130,7 @@ impl PortalStorage {
         }
 
         // Don't store if it's outside our radius
-        if self.data_radius < u64::MAX {
+        if self.data_radius < Distance::MAX {
             Ok(self.distance_to_content_id(&content_id) < self.data_radius)
```

Comment:
It seems like the comment should go inside the block. I'm surprised the test is even there. Is it basically a performance optimization? Like if we've never shrunk the radius, then the radius will equal the max and we can skip the distance calculation? That seems worth a comment on the line before the `if`.

Reply:
Yeah, this check was already present, and I didn't make any change there to minimize the diff size. I can update the comments though.